pkg/pillar/cmd/zedkube/applogs.go

```diff
@@ -146,27 +153,53 @@ func (z *zedkube) checkAppsStatus() {
 
 	var oldStatus *types.ENClusterAppStatus
 	for _, item := range items {
-		var foundPod bool
-		aiconfig := item.(types.AppInstanceConfig)
+		wdFunc()
 
+		aiconfig := item.(types.AppInstanceConfig)
 		encAppStatus := types.ENClusterAppStatus{
 			AppUUID:    aiconfig.UUIDandVersion.UUID,
 			IsDNidNode: aiconfig.IsDesignatedNodeID,
 		}
 		contName := base.GetAppKubeName(aiconfig.DisplayName, aiconfig.UUIDandVersion.UUID)
 
+		//
+		// We're looking for two pods:
+		// 1. An existing copy of a VMs virt-launcher pod which is terminating
+		//  We use this as a starting anchor to find and detach persistent volumes
+		//  So that the app can complete failover
+		// 2. A new copy of a VMs virt-launcher pod which is starting on a new node
+		//	We use this to fill in the ENClusterAppStatus above and tell the controller
+		//	Where the app is moved to.
+		//
+		// Both Pods will be of the pattern <appname>-<uuid prefix>-<pod uuid suffix>
+		terminatingVirtLauncherPod := ""
+		terminatingNodeName := ""
+		appDomainNameLbl := ""
+		foundNewSchedulingPod := false
+		var durationTerminating time.Duration
+
 		for _, pod := range pods.Items {
 			contVMIName := "virt-launcher-" + contName
-			log.Functionf("checkAppsStatus: pod %s, cont %s", pod.Name, contName)
+			log.Noticef("checkAppsStatus: pod %s, looking for cont %s", pod.Name, contName)
```

Comment:
Could all these Noticef logs produce too many log messages? I do not know how often this checkAppsStatus function is called.